src/storage_manifests.py

```diff
@@ -142,3 +143,23 @@ def evaluate(self) -> Optional[str]:
             if not self.config.get(prop):
                 return f"Storage manifests waiting for definition of {prop}"
         return None
+
+class UpdateControllerPlugin(Patch):
+    """Update the controller args in Deployments."""
+
+    def __call__(self, obj):
+        """Update the controller args in Deployments."""
+        if not any(
+            [
+                (obj.kind == "Deployment" and obj.metadata.name == "csi-cinder-controllerplugin"),
+            ]
+        ):
```

Comment:
It feels like this is easier to read

```suggestion
        if not (obj.kind == "Deployment" and obj.metadata.name == "csi-cinder-controllerplugin"):
```

Reply:
thanks!